src/test/java/org/folio/bulkops/controller/BulkOperationControllerTest.java

```diff
@@ -97,6 +98,10 @@ void shouldDownloadFileWithPreview(FileContentType type, org.folio.bulkops.domai
           .headers(defaultHeaders())
           .contentType(APPLICATION_JSON))
         .andExpect(status().isOk());
+
+      if (PROPOSED_CHANGES_FILE.equals(type) && INSTANCE_MARC.equals(entityType)) {
```

Comment:
Hi @siarhei-charniak , looks equals is used for enums, it is better to use == in such cases.